script/upload_to_aptly

```diff
@@ -12,31 +12,41 @@ set -exu
 # DOESN'T HAVE A SIGNING SUBKEY.
 #
 # Date          Channel       Fingerprint           Full Key
-#
 # 06/26/2019    Release       0x0BB75829C2D4E821    D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821
 # 12/29/2018    Beta          0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 # 12/29/2018    Dev           0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 # 12/29/2018    Nightly       0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 
 ### The first argument is the S3 bucket, the second is the GPG key fingerprint
-OPTION=${1:-brave-browser-apt-staging-nightly}
-GPG_KEY_ID=${2:-D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821} # Defaults to New Brave Software signing key
+S3_BUCKET=${1}
+GPG_KEY_ID=${2}
+DIST='artful zesty yakkety xenial jessie trusty serena stretch bionic buster cosmic disco bullseye stable'
```

Comment:
can add the newly launched `eoan` till we migrate to stable

Reply:
Resolved by adding eoan